layouts/index.html

```diff
@@ -1,7 +1,6 @@
 {{ define "main" }}
 
-{{ $js := resources.Get "js/timetree.js" | js.Build "timetree.js"  | fingerprint }}
-{{/* $js := resources.Get "js/timetree.js" | js.Build "timetree.js" | minify | fingerprint */}}
-<script src="{{ $js.RelPermalink }}"></script>
+{{ $js := resources.Get "js/timetree.js" | js.Build "timetree.js" | minify | fingerprint }}
+<script src="{{ $js.RelPermalink }}" defer></script>
```

Comment:
Yes; I had disabled the `minify` step because it makes it easier to troubleshoot problems when developing, but we want assets minified for real-world use because it's more efficient. We ideally should figure out a way to differentiate between dev and production and minify for production but not for dev (I think there's something similar for the css bundling).

Fingerprinting generates a hash for security, so the client can be sure they're downloading an uncorrupted file — see [Hugo docs on fingerprinting](https://gohugo.io/hugo-pipes/fingerprint/). However, now that I've looked I see that I'm not actually _using_ the fingerprint! Thanks for asking and prompting me to look, will remedy.

Reply:
woah! thanks for writing!! So helpful